lib/exabgp/bgp/message/update/attribute/community/extended/flowspec_scope.py

```diff
@@ -42,20 +45,13 @@ def __init__ (self, trans, asn, target, direction, community=None):
 		)
 
 	def __repr__ (self):
-		if (self.direction == 1 ):
-			str_direction = 'input'
-		elif (self.direction == 2 ):
-			str_direction = 'output'	
-		elif (self.direction == 3):
-			str_direction = 'input-output'
-		else:
-			str_direction = str(self.direction)	
+		str_direction = self.names[self.direction] 
```

Comment:
very sorry, my example was wrong, it should read:
`str_direction = self.names.get(self.direction,str(self.direction))`